tests/release/service/README.md

```diff
@@ -0,0 +1,55 @@
+# Release Service e2e tests suite
+
+Contains E2E tests related to [Release Service](https://github.com/redhat-appstudio/infra-deployments/tree/main/components/release).
```

Comment:
this should probably link to github.com/redhat-appstudio/release-service instead

Reply:
Updated.